packages/yoastseo/src/languageProcessing/helpers/sentence/SentenceTokenizer.js

```diff
@@ -42,7 +42,7 @@ export default class SentenceTokenizer {
          * \u06D4 - Urdu full stop.
          * \u061f - Arabic question mark.
         */
-		this.sentenceDelimiters = "?!\u2026\u06d4\u061f";
+		this.sentenceDelimiters = "\"?!\u2026\u06d4\u061f";
```

Comment:
We should add all alternatives for quotation marks that we have here. See `packages/yoastseo/src/languageProcessing/helpers/sanitize/quotes.js` for an overview. That would allow us to not only match `"`, but also all other possible quotation marks.

And it would be great to add `『』` to `packages/yoastseo/src/languageProcessing/languages/ja/helpers/internal/SentenceTokenizer.js` as well.